src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        Random r = new Random();
+        int n = r.nextInt(7) + 1;
+        String color = "";
+        switch (n) {
+            case (1):
+                color = "red";
+                break;
+            case (2):
+                color = "orange";
+                break;
+            case (3):
+                color = "yellow";
+                break;
+            case (4):
+                color = "green";
+                break;
+            case (5):
+                color = "light blue";
+                break;
+            case (6):
+                color = "blue";
+                break;
+            case (7):
+                color = "purple";
+                break;
+            default:
+                color = "";
+        }

```

Comment:
we can replace this huge switch-case with array of colors `private static final String[] COLORS = new String[]{"red", "green", "blue", ...}`, than get random number from 0 to COLORS.length as index and get array element of that index